### DISCUSSION TITLE
PDF Download INV Layout

### DISCUSSION THREAD

**Discussion by reaksiproton** [boxbilling/boxbilling#1399] [Q&A]
hello. can anyone tell me here how to change the pdf download invoice layout ?

**Comment by reaksiproton** [comment]
![image](https://user-images.githubusercontent.com/112498069/188781525-a3ab9a3c-b8ce-40ac-a1c1-dd8de8e3387c.png)
how to change this ?


i prefer to use :
![image](https://user-images.githubusercontent.com/112498069/188781614-3a3eb513-dd05-4213-9341-6712ce0761e3.png)
this format

**Comment by reaksiproton** [comment]
anyone have information about changing the layout?

**Reply by yagiz-dev** [reply]
AFAIK, the layout is hardcoded.

**Reply by reaksiproton** [reply]
oh my god.. then it will little difficult to edit it

but where is the template take places ?
